src/libraries/Core/Microsoft.Agents.Core/Errors/ExceptionParser.cs

```diff
@@ -68,8 +68,12 @@ private static void FormatExceptionMessage(string source, string targetSite, str
             sw.AppendLine("TimeStamp: " + DateTime.UtcNow.ToString("o", CultureInfo.InvariantCulture));
             sw.AppendLine("Error: " + message);
             sw.AppendLine($"HelpLink Url: {helpLink}");
+#if DEBUG
```

Comment:
I dont think this is the right fix. Many customers will deploy with DEBUG builds, and their stacks will be pushed to the users. 

I will comment those lines out until we have the setting, or reuse IsDevelopment (that is just an envvar)

Reply:
As I said in the comment, we will come back an make this configurable. 
IsDevelopment or anything related to Asp.net is not available here, nor is it expected to be here.

Reply:
IsDevelopment is just an envvar: `"ASPNETCORE_ENVIRONMENT": "Development"`

Publishing the Stack Trace to the end user is not good IMHO, and we cannot say: sorry, you did not build on release.

Reply:
@rido-min As I said in the previous comment and in the code, 
I will come back and make it a proper setting, this module (where this code is) is the shared module that is used outside of the hosting packages.